keyboards/opendeck/info.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
This file should be deleted.

Reply:
https://beta.docs.qmk.fm/developing-qmk/qmk-reference/reference_info_json

> You can create info.json files at every level under qmk_firmware/keyboards/<name> to specify this metadata. These files are combined, with more specific files overriding keys in less specific files. This means you do not need to duplicate your metadata information.

I disagree. This will remove duplication when other models are added.